django/contrib/admin/checks.py

```diff
@@ -130,7 +130,10 @@ def _check_fieldsets_item(self, cls, model, fieldset, label):
                     id='admin.E011',
                 )
             ]
-
+        elif not isinstance(fieldset[1]['fields'],(list,tuple)):
+            # user missed the ',' following field name tuple probably
+            return must_be('a list or tuple',option=label, obj=cls,id='admin.E008')
```

Comment:
Hi, thanks for the patch. Please use proper pep8 spacing:

`elif not isinstance(fieldset[1]['fields'], (list, tuple)):`
`return must_be('a list or tuple', option=label, obj=cls, id='admin.E008')`